### ISSUE TITLE
Add gitignore file with .DS_Store

### ISSUE THREAD

**Issue by terrifricker** [chingu-voyages/v42-toucans-team-03#53]
The folder structure in the repository is now different than what shows in my local repository.

GitHub Repository Structure
![Screen Shot 2023-01-28 at 11 06 10 AM](https://user-images.githubusercontent.com/51532684/215286335-fc7f0d7c-0a7f-4231-9e92-02435a8969c1.png)

My local structure
![Screen Shot 2023-01-28 at 11 07 20 AM](https://user-images.githubusercontent.com/51532684/215286396-58164c0d-002f-479a-84d5-a5dd6c65834b.png)

**Comment by terrifricker** [comment]
This doesn't fix the file system, but at least I'll feel comfortable with not deleting my .DS_Store files all of the time.  Alexandria looked into it and found this article about case-insensitive systems: https://www.reddit.com/r/github/comments/pmxe18/question_in_my_repo_on_github_there_are_duplicate/
And I found a similar one:
https://www.hanselman.com/blog/git-is-casesensitive-and-your-filesystem-may-not-be-weird-folder-merging-on-windows

But I haven't figured out how to actually fix it from my computer.

**Comment by terrifricker** [comment]
I've found a way to fix it.  There is now a new beta GitHub Web-based Editor.  The information is here:
https://docs.github.com/en/codespaces/the-githubdev-web-based-editor

You are able to make changes right in the browser.  I had confirmed that the uppercase Fonts folder could be completely deleted because all the font files are contained in the lowercase fonts folder.  I was able to delete the folder right in the browser and did - making a commit that went straight into development. I shouldn't have made such a huge change on my own.

Fixing the images folders includes moving a file as well as deleting a folder.  I won't do this on my own.  I'll discuss it in our main chat.
